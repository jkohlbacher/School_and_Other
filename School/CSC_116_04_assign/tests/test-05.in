resoa
heade
diffi
themo
perhu
delta
flyee
tangi
ibleo
aeiou

