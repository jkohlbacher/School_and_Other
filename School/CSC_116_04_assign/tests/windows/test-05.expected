Read 178691 words.
Tiles: [resoa]
Solution: 1 [arose] Score [5]
Tiles: [heade]
Solution: 1 [ahed] Score [8]
Solution: 2 [hade] Score [8]
Solution: 3 [haed] Score [8]
Solution: 4 [head] Score [8]
Solution: 5 [heed] Score [8]
Tiles: [diffi]
Solution: 1 [diff] Score [11]
Tiles: [themo]
Solution: 1 [home] Score [9]
Solution: 2 [meth] Score [9]
Solution: 3 [moth] Score [9]
Solution: 4 [them] Score [9]
Tiles: [perhu]
Solution: 1 [hep] Score [8]
Solution: 2 [hup] Score [8]
Solution: 3 [peh] Score [8]
Tiles: [delta]
Solution: 1 [dealt] Score [6]
Solution: 2 [delta] Score [6]
Solution: 3 [lated] Score [6]
Tiles: [flyee]
Solution: 1 [fley] Score [10]
Tiles: [tangi]
Solution: 1 [giant] Score [6]
Tiles: [ibleo]
Solution: 1 [obeli] Score [7]
Tiles: [aeiou]
Solution: 1 [eau] Score [3]
Tiles: []
